.circleci/config.yml.in

```diff
@@ -834,6 +834,8 @@ jobs:
             conda install -v -y -c file://$HOME/workspace/conda-bld torchaudio
             conda install -y pandoc 'ffmpeg<5'
             apt update -qq && apt-get -qq install -y git make
+            conda install -y gxx_linux-64
+            conda install -y cmake
```

Comment:
what do you need cmake for? and the official way to install cmake is pip.

Reply:
cmake is also for installing `pesq` library.